src/tribler-core/tribler_core/components/metadata_store/restapi/metadata_endpoint_base.py

```diff
@@ -36,10 +37,10 @@
 
 
 class MetadataEndpointBase(RESTEndpoint):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, metadata_store: MetadataStore,  *args, tags_db: TagDatabase = None, **kwargs):
```

Comment:
**Nit**: there are two spaces after `Metadatastore,`, you could remove one.

Reply:
Changed. For some reason, the linter did not catch that :thinking: